pages/docs/cloud.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: "Grouparoo Cloud Integration"
+date: "2021-11-08"
+pullQuote: "An easy-to-follow guide on integrating your Code Config repository with Grouparoo Cloud"
+---
+
+We're excited to share our Cloud offering! This guide should cover everything you need to get integrated
+using your own repository with Grouparoo Cloud.
+
+### Step 1 - Acquiring an API Key
+
+As of right now you need to make a [Trial Request](/trial) in order to get Grouparoo Cloud access. Our Administrators should get back to you within a day to help get you set up in our system and ready to integrate! After the setup process you should have an API Key and a Project ID which are required for integration.
+
+### Step 2 - Integrating your Code Config repository with Grouparoo Cloud
+
+If you do not have a Code Config repository set up, please follow our [Docs](/docs/getting-started) on how to do so.
```

Comment:
This doesn't quite line up with the sequence of events- The repository should be created, then the action added, and _then_ we can upload the grouparoo project

Reply:
This comment is really in here to tell them if they've gotten off-track.